### PR TITLE
ADEN-4991 Test ad refreshing

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  private static final String PATH = "SyntheticTests/Slots/RefreshOnView";
+
+  @Test(groups = {"AdsRefreshingFMR"})
+  public void floatingMR() {
+    AdsBaseObject page = new AdsBaseObject(driver, urlBuilder.getUrlForPath(WIKI_NAME, PATH));
+    scrollToSeeFMR(AdsContent.FLOATING_MEDREC, page);
+    page.verifyAdChainForSlot(CREATIVE_ID_CHAIN, AdsContent.FLOATING_MEDREC, page);
+  }
+
+  private void scrollToSeeFMR(String slotName, AdsBaseObject page) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java]: MAJOR Remove this unused private "scrollToSeeFMR" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+@Test(groups = "AdsRefreshing")
+public class TestAdsRefreshing extends TemplateNoFirstLoad {
+
+  private static final String[] CREATIVE_ID_CHAIN = {"108480236412", "108480237132", "108480238572", "108480239052",
+          "108480239532"};
+  private static final String WIKI_NAME = "project43";
+  private static final String PATH = "SyntheticTests/Slots/RefreshOnView";
+
+  private void scrollToSeeFMR(String slotName, AdsBaseObject page) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java]: MAJOR Remove this unused private "scrollToSeeFMR" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Comment (dianafa) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java]: it's used... :woman_facepalming:

File: src/test/java/com/wikia/webdriver/common/WindowSize.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.wikia.webdriver.common;
+
+import org.openqa.selenium.Dimension;
+
+public class WindowSize {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/WindowSize.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java [hunk 4]
```diff
@@ -8,7 +9,6 @@
 import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VideoFanTakeover;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
-
 import org.openqa.selenium.Dimension;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java]: MINOR Remove this unused import 'org.openqa.selenium.Dimension'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 2 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java [hunk 5]
```diff
@@ -188,7 +189,7 @@ private AdsDataProvider() {
             "FFFFFF"
         }, {
             WIKI_REGULAR, "Skin",
-            new Dimension(1920, 1080),
+            WindowSize.DESKTOP,
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java]: :heart:

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java (hunk 2 above)
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java]: Yep, [here](https://github.com/Wikia/selenium-tests/pull/1775/files#diff-d267aa4f0541dd396f6036b9ba857973R27) :smile_cat:

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**